pkg/pillar/types/cipherinfotypes.go

```diff
@@ -10,54 +10,81 @@ import (
 	zcommon "github.com/lf-edge/eve/api/go/evecommon"
 )
 
-// CipherContext : a pair of device and controller certificate
-// published by controller along with some attributes
-// part of EdgeDevConfig block, received from controller
-type CipherContext struct {
+// EncryptionBlock - This is a Mirror of
+// api/proto/config/acipherinfo.proto - EncryptionBlock
+// Always need to keep these two consistent.
+type EncryptionBlock struct {
+	DsAPIKey          string
+	DsPassword        string
+	WifiUserName      string // If the authentication type is EAP
+	WifiPassword      string
+	ProtectedUserData string
+}
```

Comment:
Why did you move this from further down in the file?
Just makes it harder for a reviewer to find the substance.